<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NiftyViewer</class>
 <widget class="qMRMLWidget" name="NiftyViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>426</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="status_checked">
     <property name="text">
      <string>Checked: 0 / N</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Likert Score</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="3" column="1">
       <widget class="QGroupBox" name="buttongroup">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Likert Score:</string>
        </property>
        <widget class="QRadioButton" name="radioButton_1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>1 - Acceptable,  no changes</string>
         </property>
         <property name="shortcut">
          <string>Shift+Q</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>2 - Acceptable,  minor changes</string>
         </property>
         <property name="shortcut">
          <string>Shift+W</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>3 - Unacceptable,  major changes</string>
         </property>
         <property name="shortcut">
          <string>Shift+E</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>4 - Unacceptable,  not visible </string>
         </property>
         <property name="shortcut">
          <string>Shift+R</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>358</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>5 - Bad image</string>
         </property>
         <property name="shortcut">
          <string>Shift+T</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="comment">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>125</y>
           <width>271</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="status_checked_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="save_and_next">
        <property name="text">
         <string>Save Rating, Open Next</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="overwrite_mask">
     <property name="text">
      <string>Save Mask </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
